pkg/server/plugin/datastore/sql/sql.go

```diff
@@ -534,6 +768,16 @@ func (ds *SQLPlugin) openDB(cfg *configuration) (*gorm.DB, error) {
 	return db, nil
 }
 
+// addCommonLabels append common labels for this entity to the input labels,
+// and return the result
+func (ds *SQLPlugin) addCommonLabels(labels ...telemetry.Label) []telemetry.Label {
+	return append(labels, telemetry.Label{Name: telemetry.DatabaseType, Value: ds.db.databaseType})
```

Comment:
I'm not sure how useful this label is. A single cluster of SPIRE servers is going to have the same database backend so unless you are mixing metrics from multiple clusters (which I'm not sure you'd do) this seems like overkill.